src/main/python/ayab/machine.py

```diff
@@ -43,6 +43,16 @@ def width(self) -> Literal[112, 200]:
         else:
             return 200
 
+    @property
+    def name(self) -> str:
+        if self == Machine.KH910_KH950:
+            return "KH-910, KH-950i"
+        if self == Machine.KH900_KH930_KH940_KH965:
+            return "KH-900, KH-930, KH-940, KH-965i"
+        if self == Machine.KH270:
+            return "KH-270"
+        return ""
+
```

Comment:
can we maybe factor these strings out so we aren't duplicating them in the `add_items` method below